test-case/check-suspend-resume.sh

```diff
@@ -48,7 +48,7 @@ if [ "$type" ]; then
     # check for type value effect
     grep -q "$type" /sys/power/mem_sleep || {
         grep -H '^' /sys/power/mem_sleep
-        skip_test "Unsupported sleep type argument: $type"
+        die "Unsupported sleep type argument: $type"
```

Comment:
Two changes on the same thing? Feels odd but okay...

Reply:
The first commit here is a no change cleanup that will stay. The second one switches the SKIP to a FAIL, it could be reverted if it is found to break "backwards compatibility" somewhere.

Reply:
What does the following line do here?
`grep -H '^' /sys/power/mem_sleep`

Reply:
It's just a very fancy `cat`; try it on any file and you will see every line is prefixed by the file name.